src/librbd/ImageCtx.cc

```diff
@@ -87,6 +87,15 @@ class SafeTimerSingleton : public SafeTimer {
   }
 };
 
+struct C_ThrottlingTask : public Context {
```

Comment:
@dillaman, the iops limit is based on per image, so I put the code there. I can move to ImageRequestWQ if that's a better place.